www/index.html

```diff
@@ -62,7 +62,7 @@
     <script src="manual_lib/enketo/enketo-bundle.js"></script>
     <!-- Manually installed javascript for libraries that are not published through bower -->
     <!-- cordova script (this will be a 404 in the browser) -->
-    <script type="text/javascript" src="cordova.js"></script>
+    <script src="cordova.js"></script>
```

Comment:
you don't need to revert this change in the dev branch. I have already done so in the deployment branch. Keep it around and enjoy the benefits of live reload.

Reply:
I still got this part in my code. I only do it to make the remote clean.

Reply:
I think it is better to keep these changes in the branch. There is a separate deployment branch where I pull the changes before pushing to the channel and I have removed this part on that branch. This way, if anybody else wants to start developing in the branch, they will have live reload on by default.

Reply:
Oh okay, so the live channel will always remove this code.

Reply:
Yup. I will ensure that the live channel will not have this code; you can remove it from yours.